config/log4j.properties

```diff
@@ -10,3 +10,9 @@ log4j.logger.org.apache.kafka=ERROR, stdout
 log4j.logger.org.I0Itec.zkclient=ERROR, stdout
 log4j.additivity.kafka.server=false
 log4j.additivity.kafka.consumer.ZookeeperConsumerConnector=false
+
+log4j.appender.file=org.apache.log4j.DailyRollingFileAppender
```

Comment:
one note is that with RFA + extras it can be configured to use time-based rolling policy to produce the log naming scheme (by day).  However this setup does not allow max backup index and max file size as the normal RFA provides.  Let's go with the normal RFA setup for now w/o time-based naming scheme